diagnostics/physics/sss_eval.py

```diff
@@ -14,62 +14,54 @@
+    model_grid = xarray.open_dataset( config['model_grid'] )
     target_grid = model_grid[['geolon', 'geolat']].rename({'geolon': 'lon', 'geolat': 'lat'})
     model_ave = model.mean('time').load()
 
-    # Open 3 different regional climatology datasets and merge together.
-    # These are decadal average files, except the most recent decade
-    # has been extended, so calculate a weighted average in time.
-    rcdir = '/net2/acr/regional_climatologies/'
-
-    sw = xarray.open_mfdataset([os.path.join(rcdir, 'swa_95A4_s00_10.nc'), os.path.join(rcdir, 'swa_A5B7_s00_10.nc')], decode_times=False)
-    weights = xarray.DataArray([2004-1995+1, 2017-2005+1], dims=['time'], coords={'time': sw.time})
-    sw = sw.weighted(weights).mean('time')
-
-    gom = xarray.open_mfdataset([os.path.join(rcdir, 'gom_95A4_s00_10.nc'), os.path.join(rcdir, 'gom_A5B7_s00_10.nc')], decode_times=False)
-    weights = xarray.DataArray([2004-1995+1, 2017-2005+1], dims=['time'], coords={'time': gom.time})
-    gom = gom.weighted(weights).mean('time')
-
-    nwa = xarray.open_mfdataset([os.path.join(rcdir, 'nwa_95A4_s00_10.nc'), os.path.join(rcdir, 'nwa_A5B7_s00_10.nc')], decode_times=False, combine='nested', compat='override')
-    weights = xarray.DataArray([2004-1995+1, 2017-2005+1], dims=['time'], coords={'time': nwa.time})
-    nwa = nwa.weighted(weights).mean('time')
+    # Verify that xh/yh are set as coordinates, then make sure model coordinates match grid data
+    model_grid = model_grid.assign_coords( {'xh':model_grid.xh, 'yh':model_grid.yh } )
+    model_ave = xarray.align(model_grid, model_ave,join='override')[1]
 
+    # Open regional climatology datasets and merge together.
     # Convert to a common grid to merge the datasets together.
     # concat each dataset onto a new dimension,
     # then average over the dimension.
     # with no overlap, only one region has data at any point, 
     # so mean ignores the nans from other regions and selects the available data.
-    regional_grid = {'lat': np.arange(10.05, 65.10, 0.1), 'lon': np.arange(-97.95, -39.90, 0.1)}
+    regional_grid = {'lat': np.arange( config['lat']['south'], config['lat']['north'], 0.1), 'lon': np.arange(config['lon']['west'], config['lon']['east'], 0.1)}
     # For plotting later
-    regional_gridc = {'lat': np.arange(10.00, 65.15, 0.1), 'lon': np.arange(-98.0, -39.85, 0.1)}
-    nwa_i = xesmf.Regridder(nwa, regional_grid, method='bilinear', unmapped_to_nan=True)(nwa.s_an.isel(depth=0))
-    sw_i = xesmf.Regridder(sw, regional_grid, method='bilinear', unmapped_to_nan=True)(sw.s_an.isel(depth=0))
-    gom_i = xesmf.Regridder(gom, regional_grid, method='bilinear', unmapped_to_nan=True)(gom.s_an.isel(depth=0))
-    combined = xarray.concat([nwa_i, sw_i, gom_i], dim='region').mean('region')
+    regional_gridc = {'lat': np.arange( config['lat']['south']-0.05, config['lat']['north']+0.05, 0.1), 'lon': np.arange(config['lon']['west']-0.05, config['lon']['east']-0.05, 0.1)}
```

Comment:
This won't produce the right edge values for some arrays, depending on the start and stop values passed to arange. Instead I think we could modify the `center_to_outer` function already in plot_common to also work with numpy arrays:
```python
def center_to_outer(center, left=None, right=None):
    """
    Given an array of center coordinates, find the edge coordinates,
    including extrapolation for far left and right edge.
    """
    if hasattr(center, 'values'): # handle xarray dataarrays and similar objects
        edges = 0.5 * (center.values[0:-1] + center.values[1:])
    else:
        edges = 0.5 * (center[0:-1] + center[1:])
    if left is None:
        left = edges[0] - (edges[1] - edges[0])
    if right is None:
        right = edges[-1] + (edges[-1] - edges[-2])
    outer = np.hstack([left, edges, right])
    return outer
```

And then replace this line with something like
```python
regional_gridc = {'lat': center_to_outer(regional_grid['lat']), 'lon': center_to_outer(regional_grid['lon'])}
```

Or, since `regional_gridc` is only used once, we could just directly do
```python
grid[1].pcolormesh(center_to_outer(regional_grid['lon']), center_to_outer(regional_grid['lat']), combined, transform=proj, **common)
```